client/dashboard/sites/staging-site-sync-dropdown/index.tsx

```diff
@@ -41,6 +41,18 @@ export default function StagingSiteSyncDropdown( {
 
 	const productionSiteId = site ? getProductionSiteId( site ) : null;
 	const stagingSiteId = site ? getStagingSiteId( site ) : null;
+	const siteHasManageOptions = ( site: Site ) => site?.capabilities?.manage_options;
+
+	const { data: stagingSite } = useQuery( {
+		...siteByIdQuery( stagingSiteId ?? 0 ),
+		refetchInterval: ( query ) => {
+			return siteHasManageOptions( query.state.data ) ? false : 5000;
+		},
```

Comment:
I added a refetch here because on some of my sites the Sync button didn't appear even after a long time. I realized that this query might not be refetched and this could prevent the `isStagingSiteTransferInProgress` flag to be set to false eventually.